resources/js/Components/NavBar/NavBar.vue

```diff
@@ -47,15 +51,15 @@
             </a>
         </div>
         <div class="navbar-center">
-            <img class="block lg:hidden h-8 w-auto"  :src="'images/logo-mini-blue.svg'" /> 
+            <img class="block lg:hidden h-8 w-auto"  :src="'images/logo-mini-blue.svg'" />
         </div>
         <div class="navbar-end">
             <a class="btn btn-ghost btn-circle" href="https://github.com/devpt-org">
                 <GitHubIcon width="20" height="20" />
             </a>
-            <a class="btn btn-ghost btn-circle" @click="toggleDark">
-                <SunIcon v-show="this.dark" width="20" height="20" class="svg-inline--fa fa-moon fa-w-16" />
-                <MoonIcon v-show="!this.dark" width="20" height="20" class="svg-inline--fa fa-moon fa-w-16" />
+            <a class="btn btn-ghost btn-circle" @click="toggleDark()">
```

Comment:
Quando não precisamos de passar nenhuma variável de nenhum contexto externo, podemos simplificar e passar simplesmente o nome do callback (on click).

```suggestion
            <a class="btn btn-ghost btn-circle" @click="toggleDark">
```